draft-ietf-taps-interface.md

```diff
@@ -1936,14 +1936,15 @@ is no guarantee that a soft error will be signaled.
 Connection -> SoftError<>
 ~~~
 
-### Excessive retransmissions {#conn-retrans-notify}
+### Excessive Loss {#conn-excss-loss-notify}
 
-This event notifies the application of excessive retransmissions, based on a
-configured threshold (see {{conn-excss-retransmit}}). This will only happen if
-the underlying protocol stack supports reliability and, with it, such notifications.
+This event notifies the application of excessive loss events,
+e.g., indicated by non-acked packets or exessive retransmissions,
```

Comment:
```suggestion
e.g., indicated by non-acked packets or (in case of reliable data transfer) retransmissions,
```
(note: if you don't accept this suggestion, still fix the typo: s/exessive/excessive)